layouts-robocopy/barker.html

```diff
@@ -49,100 +45,65 @@
   border-bottom: 1px dashed #cdcdcd;
 }
 .race .result {
-	width: 25%;
-	padding: 10px;
+  width: 25%;
```

Comment:
we can just change this to width: 100% now that they're separate

Reply:
👌